.codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
```

Comment:
If `codecov` config isn't setting anything other than the branch then you don't need it.